pkg/operator/openshift_sdn.go

```diff
@@ -109,3 +102,95 @@ func sdnPluginName(n v1.SDNMode) string {
 	}
 	return ""
 }
+
+// controllerConfig builds the contents of controller-config.yaml
+// for the controller
+func (h *Handler) controllerConfig() (string, error) {
+	c := h.config.Spec.DefaultNetwork.OpenshiftSDNConfig
+
+	// generate master network configuration
+	ippools := []cpv1.ClusterNetworkEntry{}
+	for _, net := range h.config.Spec.ClusterNetworks {
+		ippools = append(ippools, cpv1.ClusterNetworkEntry{CIDR: net.CIDR, HostSubnetLength: net.HostSubnetLength})
```

Comment:
This looks like the old (and still valid) format. The new multiple cidr supports has an extended syntax for the HostSubnetLength, and it is a list. Actually, what is the new network cidr syntax?

Reply:
Not sure what you mean; this is the exact network configuration type as consumed by the openshift-sdn controller...